stockfish/models.py

```diff
@@ -102,6 +102,42 @@ def set_position(self, moves: List[str] = None) -> None:
             moves = []
         self.__put(f"position startpos moves {self.__convert_move_list_to_str(moves)}")
 
+    def get_board_visual(self) -> None:
```

Comment:
Please, add test for this function

P.S. I think it would be better not to print in function but return string as result.